test/CLITest.ts

```diff
@@ -187,4 +187,49 @@ describe('CLI', function() {
       '3 + 4;'
     );
   });
+
+  it('can load plugins written with ES modules by default', async function() {
+    let afile = await createTemporaryFile('a-file.js', '3 + 4;');
+    let { status, stdout, stderr } = await runCodemodCLI([afile, '-p', plugin('increment-export-default')]);
+
+    deepEqual(
+      { status, stdout, stderr },
+      {
+        status: 0,
+        stdout: `${afile}\n1 file(s), 1 modified, 0 errors\n`,
+        stderr: ''
+      }
+    );
+    strictEqual(
+      await readFile(afile, 'utf8'),
+      '4 + 5;'
+    );
+  });
+
+  it('can load plugins with multiple files with ES modules by default`', async function() {
+    let afile = await createTemporaryFile('a-file.js', '3 + 4;');
+    let pluginFile = join(__dirname, `fixtures/plugin/increment-export-default-multiple/increment-export-default.js`);
+    let { status, stdout, stderr } = await runCodemodCLI([afile, '-p', pluginFile]);
+
+    deepEqual(
+      { status, stdout, stderr },
+      {
+        status: 0,
+        stdout: `${afile}\n1 file(s), 1 modified, 0 errors\n`,
+        stderr: ''
+      }
+    );
+    strictEqual(
+      await readFile(afile, 'utf8'),
+      '4 + 5;'
+    );
+  });
+
+  it('fails when specifying --use-local-babel as there are no plugins loaded', async function() {
+    let afile = await createTemporaryFile('a-file.js', '3 + 4;');
+    let { status } = await runCodemodCLI([afile, '-p', plugin('increment-export-default'), '--use-local-babel']);
+
+    const syntaxError = 255;
```

Comment:
I get where you’re coming from with having this as `const`, but let’s just use `let`.